README.md

```diff
@@ -91,9 +91,14 @@ the Remote Monitoring WebApp
 
 ### Deploy Azure Resources
 
-1. `pcs -t remotemonitoring -s standard`
+1. `pcs -t remotemonitoring -s standard --servicePrincipalId {servicePrincipalId} --servicePrincipalSecret {servicePrincipalSecret}`

```

Comment:
to be consistent with https://docs.microsoft.com/cli/azure/create-an-azure-service-principal-azure-cli  I would rename "secret" to "password"

Reply:
Good suggestion. That will require code change more than what i would like to do at this stage. Will add to backlog.